src/mito2/src/memtable/merge_tree/tree.rs

```diff
@@ -151,12 +151,13 @@ impl MergeTree {
             })
             .unwrap_or_default();
 
-        let partitions = self.prune_partitions(&filters);
+        let mut tree_iter_metric = TreeIterMetrics::default();
```

Comment:
We might add the cost of the `read()` method itself to `tree_iter_metric`.

Reply:
fixed here https://github.com/GreptimeTeam/greptimedb/pull/3404/commits/755736d46ad4ee3a4091aaa2b592c9f8dcc1a302#diff-c49c8c96fe96144915d2d855220956e3284752035404a1a9b230934e2ec213deR170